pkg/upstream/admin-console.go

```diff
@@ -140,7 +149,7 @@ func generateNewAdminConsoleFiles(settings *UpstreamSettings) ([]types.UpstreamF
 	upstreamFiles := []types.UpstreamFile{}
 
 	deployOptions := kotsadmtypes.DeployOptions{
-		Namespace:              "default",
+		Namespace:              settings.Namespace,
```

Comment:
Please double check if this value is ok to change.

Reply:
Good callout, looks like the public version of this function is used elsewhere. Adding a check in the public method.